docs/indices.rst

```diff
@@ -35,7 +35,7 @@ Glossary
        speciality support for X-ray data formats that follow
        :term:`OGIP` standards (such as :term:`ARF` and :term:`RMF`
        files).
-       
+
    CXC
        The `Chandra X-ray Center <http://cxc.harvard.edu/>`_.
```

Comment:
Should this be 'https' now?

Reply:
It should (although the http form will be okay for a while now). There's probably a bunch of other URLs in the docs that need updating.

Reply:
Updated the http cases (left fftw as is that page still seems to be http only).